packages/interactive_media_ads/example/android/settings.gradle

```diff
@@ -17,23 +17,11 @@ pluginManagement {
 }
 
 // See https://github.com/flutter/flutter/blob/master/docs/ecosystem/Plugins-and-Packages-repository-structure.md#gradle-structure for more info.
-buildscript {
-    repositories {
-        maven {
-            url "https://plugins.gradle.org/m2/"
-        }
-    }
-    dependencies {
-        classpath "gradle.plugin.com.google.cloud.artifactregistry:artifactregistry-gradle-plugin:2.2.1"
-    }
-}
-
 plugins {
     id "dev.flutter.flutter-plugin-loader" version "1.0.0"
     id "com.android.application" version "8.5.1" apply false
     id "org.jetbrains.kotlin.android" version "1.7.10" apply false
```

Comment:
Looks like I missed this one in my PR (I think I was only looking for the pre-declarative pattern in my grep); if you can update it here that would be great, otherwise I can do a separate PR later.

Reply:
done